theora/PSPBUILD

```diff
@@ -0,0 +1,36 @@
+pkgname=theora
+pkgver=1.2.0git
+pkgrel=1
+pkgdesc="Standard encoder and decoder library for the Theora video compression format"
+arch=('mips')
+url="https://www.theora.org/"
+license=('BSD')
+depends=('libogg')
+makedepends=()
+optdepends=()
+source=("git+https://gitlab.xiph.org/xiph/theora.git#commit=7180717276af1ebc7da15c83162d6c5d6203aabf")
```

Comment:
Do you need to use an in-dev version? Otherwise I would use 1.1.0.

Reply:
I tried building v1.1.1 and v1.2.0alpha1, but both are very outdated. I think I can get them built if I apply patches from git master branch, but that seems like it's more work than just simply building the source from git.

Reply:
That makes sense, no problem. Just leave it as it is right now then.